diff -ruN a/src/platform/linux/singleton_process.cpp b/src/platform/linux/singleton_process.cpp
--- a/src/platform/linux/singleton_process.cpp
+++ b/src/platform/linux/singleton_process.cpp
@@ -2,6 +2,7 @@
 
 #include <cerrno>
 #include <chrono>
+#include <cstring>
 #include <fcntl.h> /* For O_* constants */
 #include <sys/ipc.h>
 #include <sys/mman.h>
diff -ruN a/src/utility/argument_parsing/argument.cpp b/src/utility/argument_parsing/argument.cpp
--- a/src/utility/argument_parsing/argument.cpp
+++ b/src/utility/argument_parsing/argument.cpp
@@ -1,6 +1,7 @@
 #include "utility/argument_parser.hpp"
 
 #include <cstdlib>
+#include <cstring>
 #include <cxxabi.h>
 #include <iostream>
 #include <memory>
